kclvm/sema/src/resolver/import.rs

```diff
@@ -211,4 +232,85 @@ impl<'ctx> Resolver<'ctx> {
         self.ctx.filename = filename.to_string();
         self.scope = self.scope_map.get(pkgpath).unwrap().clone();
     }
+
+    pub(crate) fn check_unused_import(&mut self, pkgpath: &str) {
```

Comment:
1. The method is deeply nested, could you pls simplify it?
2. Add ut for check_unused_import.